.github/workflows/reusable_basic.yml

```diff
@@ -496,7 +496,7 @@ jobs:
         ${{matrix.static_hwloc}}
 
     - name: Build UMF
-      run: cmake --build ${{env.BUILD_DIR}} -j $(sysctl -n hw.logicalcpu)
+      run: cmake --build ${{env.BUILD_DIR}} -j $(sysctl -n hw.logicalcpu) || ( rm -rf ${{env.BUILD_DIR}}/_deps/jemalloc_targ-build && cmake --build ${{env.BUILD_DIR}} -j 1 )
```

Comment:
why? could we detect when we need to rm jemalloc?

Reply:
Building fails, because of multi-job compilation of jemalloc built from sources:
```
install: mkdir /Users/runner/work/unified-memory-framework/unified-memory-framework/build/_deps/jemalloc_targ-build/share: File exists
```
One-job compilation works OK. This directory needs to be removed before re-install to prevent this error.